Last-Update: 2012-08-04
Author: Dmitry Smirnov <onlyjob@member.fsf.org>
Forwarded: 2012-08-11
Bug-Debian: http://bugs.debian.org/675785
Bug-Libssh2: http://trac.libssh2.org/ticket/225
Description: Fixes pkg-config in libssh2-1-dev
 This patch addresses the problem causing FTBFS
 in packages linking with libssh2.

--- a/configure.ac
+++ b/configure.ac
@@ -102,7 +102,7 @@
 # Look for libgcrypt
 if test "$ac_cv_libssl" != "yes" && test "$use_libgcrypt" != "no"; then
   AC_LIB_HAVE_LINKFLAGS([gcrypt], [], [#include <gcrypt.h>])
-  LIBSREQUIRED=libgcrypt
+  LIBS="$LIBS -lgcrypt"
 fi
 
 AC_SUBST(LIBSREQUIRED)
